go.mod

```diff
@@ -3,17 +3,17 @@ module github.com/census-instrumentation/opencensus-service
 require (
 	contrib.go.opencensus.io/exporter/aws v0.0.0-20181029163544-2befc13012d0
 	contrib.go.opencensus.io/exporter/jaeger v0.1.1-0.20190430175949-e8b55949d948
-	contrib.go.opencensus.io/exporter/ocagent v0.5.0
+	contrib.go.opencensus.io/exporter/ocagent v0.5.1-0.20190621222907-f9129169651c // TODO: pin a released version
```

Comment:
just to confirm: there aren't changes to ocagent and stackdriver this is just to match the same proto version, correct?

Reply:
There're also some small bug fixes in both of them. You reminded me to upgrade the proto versions :) Just sent https://github.com/census-ecosystem/opencensus-go-exporter-ocagent/pull/74.

Reply:
Approved :)